src/Tools/Test Framework/Test Runner/src/DataDrivenTest/DataInputs/TestInputGroups.Page.al

```diff
@@ -0,0 +1,72 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.TestRunner;
+
+page 130462 "Test Input Groups"
+{
+    PageType = List;
+    ApplicationArea = All;
+    UsageCategory = Administration;
+    SourceTable = "Test Input Group";
+    CardPageId = "Test Input";
+    Caption = 'Test Inputs';
```

Comment:
```suggestion
    Caption = 'Test Inputs Groups';
```
Or should the page name be "Test Inputs" instead?

Reply:
I would keep it as it is, we can fix it afterwards. It is shown to the users.

Reply:
Users should not think about Groups, Inputs is sufficient.